pseudocode.md

```diff
@@ -1,6 +1,41 @@
 # Hangman (Pseudocode)
 
 ```
-Write pseudocode here, within the tripple backticks.
+Computer draws hanger(import turtle)
+Computer gets random word(import random)
+Computer gets user input for guessing the word or a letter. 
+if the letter is right = reveal where letter(s) is in the word. 
+if the word is right = reveal the word and print("Congratulations, you won!|)
+turn 1:
+if the letter is not in the word = draw a head connected to hanger. 
+or
+if the word is wrong = draw a head connected to hanger.
+turn 2:
+if the letter is not in the word = draw the body. 
+or
+if the word is wrong = draw the body
+turn 3:
+if the letter is not in the word = draw the right arm connected to hanger. 
+if the word is wrong = draw the right arm connected to hanger. 
+turn 4:
+if the letter is not in the word = draw the left arm connected to hanger. 
+if the word is wrong = draw the left arm connected to hanger.
+turn 5:
+if the letter is not in the word = draw the right leg connected to hanger. 
+if the word is wrong = draw the right leg connected to hanger.
+turn 6:
+if the letter is not in the word = draw the left leg connected to hanger. 
+if the word is wrong = draw the left leg connected to hanger.
+
```

Comment:
Good detail on the different cases. This will come in handy when deciding which image to draw for the hangman's platform.